src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVParser.scala

```diff
@@ -10,16 +10,16 @@ package org.broadinstitute.dsde.firecloud.utils
 case class TSVLoadFile(
   firstColumnHeader:String, //The first header column, used to determine the type of entities being imported
   headers:Seq[String], //All the headers
-  tsvData:Seq[Map[String, String]] //List of rows of the TSV. Each row is a map from header to value.
+  tsvData:Seq[Seq[String]] //List of rows of the TSV, broken out into fields
 )
```

Comment:
1. I needed to know the reference type, too, in addition to the column name when that's defined, so the info was incomplete.  (Yes, I could do a lookup while processing every field, but that's silly, because it can be done once for all time for each column and saved ahead of time.)
2. It's crazy extravagant with memory to zip column info with every field up front rather than to do it row by row on demand.